pages/index.vue

```diff
@@ -479,9 +487,14 @@ function handleImageAdd() {
             <UButton class="m-1" @click="handleImage" :color="upImages.length?'primary':'gray'"
                      icon="i-heroicons-photo"/>
           </UTooltip>
-          <UButton @click="handleReq(selectedModel)" :disabled="loading" class="m-1">
+          <UButton @click="handleReq($event, selectedModel)" :disabled="loading" class="m-1">
             发送
           </UButton>
+          <UButton color="gray" size="md" class="m-1" @click="() => input = ''">
```

Comment:
Can you remove the clear button? I think it's not necessary.

Reply:
Sure~